terraform/modules/ci/main.tf

```diff
@@ -35,10 +39,41 @@ module "server_service" {
   disable_dedicated_sa = true
 }
 
-resource "google_cloud_run_service" "client_services" {
+resource "google_cloud_run_service" "ingestion_client_services" {
```

Comment:
Since we're making this change, can we make the names more intuitive? Like

`ingestion_backed_services` and `cloudlogging_backed_services`. Especially for `cl` my first idea is changelist rather cloud logging.

Reply:
done